src/env/prod/terraform.tfvars

```
@@ -292,6 +293,44 @@ db_metric_alerts = {
     dimension   = []
   }
 }
+
+cosmos_mongo_db_params = {
+  enabled      = false
+  kind         = "MongoDB"
+  capabilities = ["EnableMongo", "EnableServerless"]
```

```suggestion
  capabilities = ["EnableMongo"]
```
serverless is for dev/test envs. For production with heavy loads better use provisioned throughput